lightning/src/ln/functional_tests.rs

```diff
@@ -2352,11 +2352,11 @@ fn test_justice_tx() {
 	let mut alice_config = UserConfig::default();
 	alice_config.channel_handshake_config.announced_channel = true;
 	alice_config.channel_handshake_limits.force_announced_channel_preference = false;
-	alice_config.channel_handshake_config.our_to_self_delay = 6 * 24 * 5;
+	alice_config.channel_handshake_config.our_to_self_delay = 1008 + 288;
 	let mut bob_config = UserConfig::default();
 	bob_config.channel_handshake_config.announced_channel = true;
 	bob_config.channel_handshake_limits.force_announced_channel_preference = false;
-	bob_config.channel_handshake_config.our_to_self_delay = 6 * 24 * 3;
+	bob_config.channel_handshake_config.our_to_self_delay = 1008 + 144;
```

Comment:
Note to reviewers, the value there has been bumped to maintain the original asymmetry between alice and bob config. From my memory authoring this test the purpose was testing we correctly claim justice transaction on revoked output on counterparty transaction where the counterparty request a different value than ours.